package.json

```diff
@@ -56,6 +56,7 @@
     "karma-safari-launcher": "^1.0.0",
     "karma-webpack": "^2.0.3",
     "mocha": "^3.4.2",
+    "nyc": "^11.0.2",
```

Comment:
@belochub nope, I checked it out. It only includes `nyc@10.3.2` while this PR requires `nyc@11`.